http-body-util/Cargo.toml

```diff
@@ -27,7 +27,7 @@ categories = ["web-programming"]
 
 [dependencies]
 bytes = "1"
-futures-util = { version = "0.3", default-features = false, features = ["alloc"] }
+futures-util = { version = ">=0.3.14, <0.4.0", default-features = false, features = ["alloc"] }
```

Comment:
```suggestion
futures-util = { version = "0.3.14", default-features = false, features = ["alloc"] }
```

should be the same

Reply:
Yea, what David suggested would be best. futures-util _does_ follow semver, it is fine to export a trait in an additional place in a newer version.

Reply:
Updated.

I don't think this is following semver, though — it's fine to export a trait in a additional place, what's not fine is that they stopped exporting it in the old place — that's a breaking change, since code that previously referenced that name will no longer compile.

Reply:
I mean, I guess technically since it's a `0.x.x` release that would indicate that they make no promises about API compatibility, which would point to locking to a specific version. I'm not sure if the futures-rs maintainers intend to offer any guarantees beyond that, though, it wouldn't surprise me if they did.

Reply:
Is that the case? I could be confused, but it seems to me: http-body wanted the trait at a certain path. It is there in futures-util 0.3.14, but not 0.3.13, which was older.

Reply:
Ah, yeah, it seems you're correct — I looked at the futures-rs code but must have missed where the `futures_util::Future` export was, since I didn't see it in the current code, but I just went and tested it and you're correct. Will update the commit message.

Reply:
Updated the message.